editor/gui/editor_file_dialog.cpp

```diff
@@ -56,6 +58,97 @@ EditorFileDialog::GetIconFunc EditorFileDialog::get_thumbnail_func = nullptr;
 EditorFileDialog::RegisterFunc EditorFileDialog::register_func = nullptr;
 EditorFileDialog::RegisterFunc EditorFileDialog::unregister_func = nullptr;
 
+void EditorFileDialog::popup(const Rect2i &p_rect) {
```

Comment:
There is lots of duplication between this and the method below. You could try calling `popup()` in `set_visible()`, or at least extract the duplicated parts into separate method.